tests/test.c

```diff
@@ -326,6 +327,7 @@ struct test tests[] = {
         _TEST(0098_consumer_txn, 0),
         _TEST(0099_commit_metadata, 0),
         _TEST(0100_thread_interceptors, TEST_F_LOCAL),
+        _TEST(0101_fetch_from_follower, TEST_BRKVER(2,4,0,0)),
```

Comment:
note: need to comment out `TEST_BRKVER(2,4,0,0)` to test against trunk (2.4 not out yet).